variants/STM32C0xx/C011J(4-6)M/generic_clock.c

```diff
@@ -18,10 +18,37 @@
   * @param  None
   * @retval None
   */
-WEAK void SystemClock_Config(void)
+
+void SystemClock_Config(void)
```

Comment:
Keep it WEAK to allow redefinition if needed
```suggestion
WEAK void SystemClock_Config(void)
```